### PR TITLE
Add priority parameter to server.

### LINKED ISSUE #88: Priority isn't available
Any chance we can get an update so we can use the new fields like priority? https://developers.google.com/cloud-messaging/server-ref

### PR DESCRIPTION
**Pull request by kroikie**
Resolves #88 allowing user to set message priority.

### CODE REVIEW

File: client-libraries/java/rest-client/src/com/google/android/gcm/server/Constants.java [hunk 1]
```diff
@@ -63,6 +63,11 @@
   public static final String PARAM_TIME_TO_LIVE = "time_to_live";
 
   /**
+   * Prefix to HTTP parameter used to set the message priority.
```
Comment (silvolu) [inline client-libraries/java/rest-client/src/com/google/android/gcm/server/Constants.java]: This is not a prefix, this is the parameter name. Please change the docs for time to live as well.

File: client-libraries/java/rest-client/src/com/google/android/gcm/server/Message.java [hunk 2]
```diff
@@ -128,6 +131,14 @@ public Builder restrictedPackageName(String value) {
       return this;
     }
 
+    /**
+     * Sets the priority property.
+     */
+    public Builder priority(String value) {
```
Comment (silvolu) [inline client-libraries/java/rest-client/src/com/google/android/gcm/server/Message.java]: Given that there are only 2 allowed values for priority, "normal" and "high", you could create an enum and then inside the builder switch on the value and convert to the switch to the corresponding string value

**Comment by silvolu** [comment]
Ping?

File: client-libraries/java/rest-client/src/com/google/android/gcm/server/Constants.java (hunk 1 above)
Comment (kroikie) [inline client-libraries/java/rest-client/src/com/google/android/gcm/server/Constants.java]: Done

File: client-libraries/java/rest-client/src/com/google/android/gcm/server/Message.java (hunk 2 above)
Comment (kroikie) [inline client-libraries/java/rest-client/src/com/google/android/gcm/server/Message.java]: Done

**Pull request merged**